money-core/src/main/scala/com/comcast/money/core/MoneyTracerProvider.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2012 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.core
+
+import io.opentelemetry.trace
+import io.opentelemetry.trace.TracerProvider
+
+case class MoneyTracerProvider(tracer: Tracer) extends TracerProvider {
+  override def get(instrumentationName: String): trace.Tracer = tracer
+  override def get(instrumentationName: String, instrumentationVersion: String): trace.Tracer = tracer
```

Comment:
do we want to support getting a specific version? I believe this should error out if there`s a version mismatch

Reply:
Let me check the docs on this interface to see exactly what is expected here.  Feels weird that if you're going through the trouble to set up SPI that you'd also make specific library+version requests to obtain a tracer.

Reply:
From the JavaDocs it mentions that the name/version is for the instrumentation library, I don't think Money would care about that.

Reply:
Looks like the SDK reference implementation returns a different `TracerSdk` instance for each requested instrumentation library.  It doesn't matter what the library names or versions are.  The library info is attached to the span and the exporter can inspect/use it, but that's all it does.